deploy/k8s/controller.yaml

```diff
@@ -13,6 +13,8 @@ spec:
     metadata:
       labels:
         app: che-workspace-controller
+      annotations:
+        kubectl.kubernetes.io/restartedAt: ""
```

Comment:
it's a tricky thing we can do to make sure that the deployed is retriggered after we apply `./deploy` and there is no change in deployment but is in configmap.

Instead we could just:
```
configure: _update_yamls _apply_controller_cfg _reset_yamls restart
```
but then deployment may be triggered twice if there is changes in deployment as well.

Controller has some code to reload configuration after config map is changed, but I don't think it will work with webhooks_enabled...

@amisevsk WDYT? Maybe it is the option to go and having +1 triggering of deployment is not critical at all:
```
configure: _update_yamls _apply_controller_cfg _reset_yamls restart
```
or updating restartAt is good enough to proceed with?

Reply:
Makes sense to me :+1: